lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -90,6 +90,9 @@ func RuleAPIV2ToBackend(ar apiv2.Rule, ns string) model.Rule {
 		} else {
 			srcSelector = sourceNSSelector
 		}
+	} else {
```

Comment:
would it be easier to set these values on line 78 and only overwrite it if there is a NS selector?

Reply:
yes, I considered that approach. I thought this might be easier to read, but happy to change if you'd like.